.github/workflows/trivy-security-scan.yml

```diff
@@ -30,7 +30,9 @@ jobs:
         uses: actions/cache@v4
         with:
           path: .cache/trivy
-          key: cache-trivy-${{ steps.date.outputs.date }}
+          key: trivy-${{ steps.date.outputs.date }}
+          restore-keys: |
+            trivy-
```

Comment:
This will not have the desired behavior since it will reuse the same cache over and over again without updates as long as it is not evicted from cache

Reply:
@nazar-pc I'm going to try caching with digest instead of date. That way it will only evict the cache when the digest changes (updates to the DB).

Reply:
I do not see any logic in this. The date is on our end. We check the cache of our repository, how does making additional pull of `ghcr.io/aquasecurity/trivy-db:2` on top of this helps in any way?

Reply:
It's not `pull`ing the image layers, just the manifest of the artifact. yes, i know the date is on our side obviously, but my goal was to just check if the DB is updated and if so then cache. Rather than just filling the cache every day until GH evicts it.